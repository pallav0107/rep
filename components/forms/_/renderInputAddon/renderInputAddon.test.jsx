import React from 'react';
import { shallow } from 'enzyme';
import { withTheme } from '../../../../../utils/jest/TestUtils';
import "../../../../setupTests"

describe('renderInputAddon', () => {
  it('should render', () => {
    const component = withTheme(
      <renderInputAddon meta={['touched', 'error']} />
    );
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
  });
});
