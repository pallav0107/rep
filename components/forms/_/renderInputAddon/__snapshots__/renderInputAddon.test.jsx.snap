// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderInputAddon should render 1`] = `
<renderInputAddon
  meta={
    Array [
      "touched",
      "error",
    ]
  }
/>
`;
